### ISSUE TITLE
Exception in thread Thread-1 When Using Geofence

### ISSUE THREAD

**Issue by rrbailey89** [kvangent/PokeAlarm#135]
## Your Environment

Operating System:  Windows 10
Python Version: (use `python -V`)  Python 2.7.12
PokemonGo-Map startup command: `python runserver.py -os -l "33.2241637, -117.3823953"`  
PokeAlarm startup command: `python runwebhook.py`

Did the issue persist after restarting the script?: yes

## Console Output

```
Exception in thread Thread-1:
Traceback (most recent call last):
  File "C:\Python27\lib\threading.py", line 801, in __bootstrap_inner
    self.run()
  File "F:\Ray\downloads\pogomap\pokealarm\alarms\alarm_manager.py", line 72, in run
    self.trigger_pkmn(data['message'])
  File "F:\Ray\downloads\pogomap\pokealarm\alarms\alarm_manager.py", line 109, in trigger_pkmn
    if 'GEOFENCE' in config and not config['GEOFENCE'].contains(lat,lng):
  File "F:\Ray\downloads\pogomap\pokealarm\alarms\geofence.py", line 40, in contains
    return self.polygon.encloses_point(Point(x,y))
AttributeError: 'Point' object has no attribute 'encloses_point'
```
Also see the attached logs. It seems that posts continue to happen after the exception but no longer containing any relevant information.
[Debug 1.txt](https://github.com/kvangent/PokeAlarm/files/427899/Debug.1.txt)
[Debug 2.txt](https://github.com/kvangent/PokeAlarm/files/427900/Debug.2.txt)
## Your Alarms Config

```json
		},
		{
			"active": "True",
			"type":"slack",
			"api_key":"removed",
			"channel":"#general",
			"map":{
				"enabled":"True",
				"width":"300",
				"height":"150",
				"maptype":"roadmap",
				"zoom":"16"
			}			
		},
```

## Troubleshooting Checklist
[ x] Most recent version of PokemonGo-Map
[ x] Most recent version of PokeAlarm
[ x] Checked the Wiki page for service in question

**Comment by Atitoa** [comment]
Changing line 33 in geofence.py from
self.polygon = Polygon(points) to self.polygon = Polygon(*points)
fixed it for me

**Comment by rrbailey89** [comment]
@Atitoa thanks that fixed it for m

**Comment by kvangent** [comment]
@rrbailey89 I've pushed this change so you shouldn't have to worry about it manually. It is interesting that it works fine for me (and others), but not for all cases? Especially since your python is up to date. I don't know if you are getting an older version of sympy or perhaps I am.

Also as a air warning, the new geofence implementation seems to be quite slow .I noticed a 2 second delay in processing that. If you experience issues with backup, set a distance limit to reduce the number of needed calculations.
